cdap-docs/developers-manual/source/building-blocks/artifacts.rst

```diff
@@ -0,0 +1,24 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :copyright: Copyright © 2015 Cask Data, Inc.
+
+.. _artifacts:
+
+=========
+Artifacts
+=========
+
+.. highlight:: java
+
+An *Artifact* is a JAR file that contains Java classes and resources required to create and run an *Application*.
+*Artifacts* are identified by a name, version, and scope. Once added to CDAP, an *Artifact* cannot be modified,
+unless it is a SNAPSHOT artifact. An *Artifact* is a SNAPSHOT artifact if its version begins with SNAPSHOT.
```

Comment:
yeah sorry this should be changed, it's if the version classifier starts with snapshot.  I will add some examples